nautobot/core/tests/test_graphql.py

```diff
@@ -486,36 +487,72 @@ def test_graphql_query_format(self):
         self.assertEqual(site_names, ["Site 1", "Site 2"])
 
 
-class GraphQLQuery(APITestCase):
+class GraphQLQuery(TestCase):
```

Comment:
Are we losing test coverage by dropping `APITestCase` here?

Reply:
I don't think so 
Actually we are getting more flexibility by not using the API since we have now access to the ExecutionResult object directly and we can access the errors as python Objects